apollo-sample/build.gradle

```diff
@@ -29,8 +29,9 @@ dependencies {
   compile dep.appcompat
   compile dep.rxjava2
   compile dep.rxandroid
-  compile dep.retrofitRx2Adapter
   compile dep.okhttpLoggingInterceptor
+  compile project(path: ':apollo-runtime')
```

Comment:
please depend on the maven artifact here. this way we will know if there was a problem with the published version